packages/@expo/cli/src/run/android/runAndroidAsync.ts

```diff
@@ -108,6 +110,13 @@ export async function runAndroidAsync(projectRoot: string, { install, ...options
   } else {
     await manager.stopAsync();
   }
+
+  if (shouldUpdateBuildCache && projectConfig.exp.experiments?.remoteBuildCache) {
+    await uploadRemoteBuildCache(projectRoot, {
```

Comment:
does using `await` here block anything? (not sure what the caller of `runAndroidAsync` / `runIosAsync` blocks when awaiting completion of the function)

Reply:
Not really, that's the last function called by this whole command